source/scpd/types/XDRBase.d

```diff
@@ -96,7 +96,11 @@ struct opaque_array(uint32_t N = XDR_MAX_LEN)
 {
     BitBlob!(N * 8) base;
     alias base this;
-    mixin ForwardCtors!base;
+
```

Comment:
This seems to be the last place `ForwardCtors` was still used, so we could delete this code in the `Util` module as it's no longer used.

Reply:
In fact the whole module can be nuked